website/src/content/concepts/integrate-with-your-own-api.mdx

```diff
@@ -151,6 +151,11 @@ The middleware requires some options:
 
 - `jwks` (_object_): See [options of `jwks-rsa`](https://github.com/auth0/node-jwks-rsa/).
 
+- `getRequestUrl` (_function_): Returns the URL (URI path + query string) of the `request` to validate the audience. Use this in case the `request` object does not contain either an `originalUrl` or an `url` property (see serverless example below for using AWS Lambda functions).
+
+  ```ts
+  type function getRequestUrl(request: Request): string;
```

Comment:
Missing closing code block backticks (github suggestion ignore them apparently.

Reply:
Updated in this commit e4be0ae17a4678626bf426a1c7ab237e458a00d1